### PR TITLE
Add Personal Management System to Personal Dashboards

### PR DESCRIPTION
**Pull request by Volmarg**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1160,6 +1160,7 @@ See also _[Monitoring](#monitoring)_
 - [Homer](https://github.com/bastienwirtz/homer) - A dead simple static homepage to expose your server services, with an easy yaml configuration and connectivity check. `Apache-2.0` `HTML5`
 - [iDashboard-PHP](https://github.com/causefx/iDashboard-PHP) - HTPC Dashboard to load website services. `MIT` `PHP`
 - [Organizr](https://github.com/causefx/Organizr) - Organizr aims to be your one stop shop for your Servers Frontend. `GPL-3.0` `PHP`
+- [Personal management system](https://github.com/Volmarg/personal-management-system) - Central point for managing personal data (billings, payments, job holidays, notes etc.) ([Demo](http://185.204.216.136:8000/)), `MIT` `PHP`
```
Comment (nodiscc) [inline README.md]: - Missing final dot after description
- Extra comma after demo link
- Can you host the demo at a proper domain name?

**Comment by Volmarg** [comment]
I will checkout out how the normal domain case looks like - shouldn't be a problem but just got to make small research for that.

Will also fix that coma/dot once I know how is domain going and will get back with that.

File: README.md (hunk 1 above)
Comment (SuperSandro2000) [reply README.md]: FYI the demo does not work right now for me.

Also https would be awesome 😉

Comment (Volmarg) [reply README.md]: Yes the demo died becauase i just added domain to it and i have acces only to terminal on smartphone now so had issues to configure it but works now under New url - will add it to pull in moment.

**Comment by Volmarg** [comment]
Ive added changes along with domain based demo. However https is out of option.

**Comment by nodiscc** [comment]
FYI you can get a free subdomain name at https://freedns.afraid.org/ which is fine for demo purposes.

> https is out of option.

You can also get free SSL certificates from Let's Encrypt, check https://www.digitalocean.com/community/tutorials/how-to-secure-apache-with-let-s-encrypt-on-ubuntu-18-04 (your server headers disclose the exact apache version, you should probably hide these). It's not mandatory however a demo for an application that manages financial/personal information without HTTPS sends a bad signal IMHO.

Thanks again

**Comment by Volmarg** [comment]
Thanks for info. The domain that I've just added is... also free so fine for me :D
I will check that ssl in free time and once i have access to laptop - I will have that in mind.

**Pull request merged**